src/pricing-fetcher.ts

```
@@ -31,6 +33,22 @@ export class PricingFetcher implements Disposable {
 
+		// If we're in offline mode, return pre-fetched data
+		if (this.offline) {
+			try {
+				const pricing = new Map(Object.entries(prefetchClaudeData));
+				this.cachedPricing = pricing;
+				return pricing;
+			}
+			catch {
+				// If prefetchClaudeData is not available, return empty map
+				// This allows the application to continue running without cost calculations
+				logger.warn('Offline mode requested but pre-fetched pricing data not available. Cost calculations will return 0.');
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)